### PR TITLE
feat: /ceph/install redesign

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Applied redesign as per [doc](https://docs.google.com/document/d/1EmOtHjuEMKhVLQA7GwyrKfCzOKfc8UCEyh-p04-PxzA/edit?tab=t.0) and [design](https://www.figma.com/design/ulUyOFdQ7CNy0whuJ4S2FM/Ceph-Bubble?node-id=16-4455&node-type=frame&m=dev)
- Drive by:
  - Fixed the djlint toggle setting not placed correctly on `/_latest-blogs_download.html`

## QA

- View the site locally in your web browser at: https://ubuntu-com-14511.demos.haus/ceph/install
    - Be sure to test on mobile, tablet and desktop screen sizes
- Check against doc and design 

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-16347

### CODE REVIEW

File: templates/download/shared/_latest-blogs_download.html [hunk 1]
```diff
@@ -27,8 +27,8 @@ <h3 class="p-heading--5">
 </section>
 
 <script src="{{ versioned_static('js/dist/latest-news.js') }}"></script>
+{# djlint:off #}
```
Comment (mtruj013) [inline templates/download/shared/_latest-blogs_download.html:30]: djlint will ignore this if it's set within the script

**Comment by eliman11** [comment]
Thank you! A few comments:
- [ ] Missing description for step 5 under 'multi-node deployment' tab --> 'Here you should see that all the nodes you added have joined the cluster, in the familiar ceph status output.'
- [ ] Change 'containerised' to 'containerized' in the 3rd bullet point under 'containerized' tab
- [ ] Add 'news' to 'Latest news from our blog' heading at the bottom
- [ ] The link in the blog section heading is returning a 404 error - think it's missing /tag. Could you link it to https://ubuntu.com/blog/tag/ceph please? Also edited on the copydoc

**Comment by mattea-turic** [comment]
ty @mtruj013 ! just one question from me:

- there seems to be a lot of wrapping going on here, making the padding pretty deep; could there just be a regular 64px being used?
Screenshot 2024-11-27 at 16 29 53

**Comment by mtruj013** [comment]
Thanks @mattea-turic and @eliman11, ready for another look!

**Comment by eliman11** [comment]
Looks good thank you!

**Comment by mattea-turic** [comment]
I just noticed the links under the tabs "containerized" and "large-scale"  need a hr above it, could you add that in pls? For "large-scale", if you could align them horizontally too like you would as part of a cta block so there aren't too many lines

Screenshot 2024-11-28 at 17 00 48

I'll +1 though :)

File: static/sass/styles.scss [hunk 2]
```diff
@@ -166,6 +166,7 @@ $color-shadow: rgba(0, 0, 0, 0.5) !default;
 @include ubuntu-u-vertically-center;
 @include ubuntu-p-dell;
 @include ubuntu-p-tube-line;
+@include ubuntu-p-detailed-list;
```
Comment (petesfrench) [inline static/sass/styles.scss:169]: if we are included 'ubuntu-p-lists' above, do we need this?

File: templates/ceph/install.html [hunk 3]
```diff
@@ -2,278 +2,131 @@
...
 {% block content %}
 
-<section class="p-strip--suru-topped">
-  <div class="row u-equal-height">
-    <div class="col-7">
-      <h1>Install Ceph on Ubuntu</h1>
-      <p class="u-sv2">Ceph is a storage system designed for excellent performance, reliability, and scalability. However, the installation and management of Ceph can be challenging. The Ceph-on- Ubuntu solution takes the administration minutiae out of the equation through the use of Juju charms. With charms, the deployment of a Ceph cluster becomes trivial as does the scaling of the cluster&rsquo;s storage capacity.</p>
-      <p>Looking for help running Ceph?</p>
+  {% call(slot) vf_hero(
+    title_text='Install Ceph on Ubuntu',
```
Comment (petesfrench) [inline templates/ceph/install.html:22]: Just a thought, but we should establish a consistent rule on using single or double marks in Jinja templates

File: templates/ceph/shared/_latest-news-ceph.html [hunk 4]
```diff
@@ -0,0 +1,44 @@
+<section class="p-section--deep">
+  <div class="u-fixed-width">
+    <hr class="p-rule" />
+    <div class="p-section--shallow">
+      <h2>
+        <a href="{{ title_link }}">Latest news from&nbsp;our&nbsp;blog&nbsp;&rsaquo;</a>
```
Comment (petesfrench) [inline templates/ceph/shared/_latest-news-ceph.html:6]: Are the `&nbsp;`'s needed here?

File: templates/ceph/shared/_latest-news-ceph.html [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+      </h2>
+    </div>
+  </div>
+  <div class="row p-section--shallow" id="ceph-install-latest"></div>
+
+  <template style="display:none" id="ceph-install-template">
```
Comment (petesfrench) [inline templates/ceph/shared/_latest-news-ceph.html:12]:
```suggestion
  <template id="ceph-install-template">
```
Might be wrong here, but I think `template`'a are hidden from the page flow by default

**Review by petesfrench — APPROVED** [review]
LGTM

File: templates/ceph/install.html (hunk 3 above)
Comment (mtruj013) [reply templates/ceph/install.html:22]: Agreed, we seem to use double quotes everywhere else so I'll update to that

File: static/sass/styles.scss (hunk 2 above)
Comment (mtruj013) [reply static/sass/styles.scss:169]: We do not, thanks for catching!

File: templates/ceph/shared/_latest-news-ceph.html (hunk 4 above)
Comment (mtruj013) [reply templates/ceph/shared/_latest-news-ceph.html:6]: Most of this template is copied from the /desktop version (layout was different so I couldn't reuse), but at the time of building that these were requested by design

File: templates/ceph/shared/_latest-news-ceph.html (hunk 5 above)
Comment (mtruj013) [reply templates/ceph/shared/_latest-news-ceph.html:12]: You're right, updated!

**Pull request merged**